lib/command_runner.dart

```diff
@@ -158,13 +180,32 @@ class CommandRunner<T> {
 
           command.usageException('Missing subcommand for "$commandString".');
         } else {
+          var requested = argResults.rest[0];
+
+          // Build up a help message containing similar commands, if found.
+          var similar = StringBuffer();
+          if (suggestedCommandsMaxEditDistance > 0) {
+            var possible = similarCommands(requested);
+            if (possible.isNotEmpty) {
+              similar.write(' The most similar command');
```

Comment:
How about "Did you mean one of these?"

Reply:
This is the text that `git` uses, but I agree it feels a bit awkard. I will update it.

Reply:
Updated, I also changed the formatting a bit, generally looks like this now:

```
Could not find a command named "$typo".

Did you mean one of these?
  long
```

Reply:
cc @jonahwilliams @bkonyi how do you feel about the new text?